Structural and other minor changes

Mazey opened the pull request:
-Added modules folder, moved func.js (now util.js) and server list there)
-Changed as many CamelCase variables as I could find into snake_case (snake case for variables, camel case for functions)
-Minor changes to how messages are deleted
-Removed index.js requiring util.js and util.js requiring index.js

Mazey committed:
Structural and other minor changes

-Added modules folder, moved func.js (now util.js) and server list there)
-Changed as many CamelCase variables as I could find into snake_case (snake case for variables, camel case for functions)
-Minor changes to how messages are deleted
-Removed index.js requiring util.js and util.js requiring index.js

eps0003 commented on modules/util.js:
```diff
@@ -0,0 +1,217 @@
+	});
+},
+
+exports.updateServerList = (message, servers) => {
```
Should this function be moved into the server list module?

eps0003 reviewed (commented):
Why did you delete package-lock.json?

eps0003 commented on modules/serverlist.js:
```diff
@@ -0,0 +1,35 @@
+		util.updateServerList(server_list, servers);
+
+		let total_players = servers.reduce((t, x) => t + x.currentPlayers, 0);
+		client.user.setPresence({ status: 'online', game: { name: `${total_players} in KAG | ${config.prefix}help` } });
```
This shouldn't be in the server list module, but it does need a loop to update it.

Mazey committed:
Structural improvements

-Moved update presence to util.js
-Changed "Playing x in KAG" to "Watching x in KAG"
-Moved updateServerList to serverlist.js
